source/code-snippets/usage-examples/find.js

```diff
@@ -32,7 +32,7 @@ async function run() {
       .project(projection);
 
     // print a message if no documents were found
-    if (!(await cursor.hasNext())) {
+    if ((await cursor.count()) == 0) {
```

Comment:
Should we use `===` here to be clear that we're not casting any types?

Reply:
Great point, `===` is definitely the right choice here even if `==` works to show best practice. Changing for multiple occurrences of this now.